ros_gz_bridge/README.md

```diff
@@ -31,6 +31,7 @@ The following message types can be bridged for topics:
 | geometry_msgs/msg/Twist              | ignition::msgs::Twist                       |
 | geometry_msgs/msg/TwistStamped       | ignition::msgs::Twist                       |
 | geometry_msgs/msg/TwistWithCovariance| ignition::msgs::TwistWithCovariance         |
+| gps_msgs/GPSFix                      | ignition.msgs.NavSat                        |
```

Comment:
Should we update the README and replace `ignition` with `gz`?

Reply:
Since humble and iron officially support Fortress, we've been keeping the `ignition` name here. The formatting needs to be fixed though. `::` instead of `.`.